src/components/Layout/index.tsx

```diff
@@ -54,6 +63,17 @@ export const Layout = ({ children }: LayoutProps) => {
           © {new Date().getFullYear()},{" "}
           {footerMessage[Math.floor(Math.random() * footerMessage.length)]} by{" "}
           <TextLink to="https://yordevs.com">Yordevs</TextLink>
+          <SocialLogoHolder>
+            <TextLink to="https://www.facebook.com/thelemonpress">
+              <FaFacebookF />
+            </TextLink>
+            <TextLink to="https://www.twitter.com/thelemonpress">
+              <FaTwitter />
+            </TextLink>
+            <TextLink to="https://www.instagram.com/thelemonpressyork">
+              <FaInstagram />
+            </TextLink>
+          </SocialLogoHolder>
```

Comment:
This doesn't quite work responsively yet with the site, but we can look past that for the meeting and fix it later?